package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "install": "npm run bootstrap",
     "bootstrap": "lerna bootstrap --nohoist=@aragon/os",
-    "test": "lerna run --scope=@aragon/apps-* --concurrency=1 --stream test",
+    "test": "lerna run --scope=@aragon/apps-* --scope=@aragon/templates-beta --concurrency=1 --stream test",
```

Comment:
are we sure we can pass 2 scope arguments? worried it might override it

Reply:
Yes, it worked for me. I actually added it for lint and coverage too. See first one for instance:
```
lerna success run Ran npm script 'lint' in packages:
lerna success - @aragon/apps-fundraising
lerna success - @aragon/apps-payroll
lerna success - @aragon/apps-finance
lerna success - @aragon/apps-token-manager
lerna success - @aragon/apps-vault
lerna success - @aragon/apps-voting
lerna success - @aragon/templates-beta
```